src/task2.js

```diff
@@ -0,0 +1,45 @@
+const csv = require('csvtojson');
+const fs = require('fs');
+
+const writeText = fs.createWriteStream('node_mentoring_t1_2_input_example.txt');
+const results = [];
+
+var s = fs.createReadStream('node_mentoring_t1_2_input_example.csv')
```

Comment:
Consider not using a var variables at this point since there are better es6 alternatives.

Reply:
Updated as per the comment